repeating_dates.module

```diff
@@ -59,6 +59,7 @@ function repeating_dates_field_widget_info() {
         'hide_to_date' => FALSE,
         'default_repeat_freq' => 'weekly',
         'default_stop_mode' => 'count',
+        'default_count' => 1,
```

Comment:
I don't think, a default value of "1" is really a good one - I can't think of a real use-case for that.

This means, admin have to think about that value and actively change the default, whenever creating a field of that type.

Reply:
I haven't looked at this in a while. Good point. I'll make it NULL and see.